spec/models/visit_spec.rb

```diff
@@ -28,6 +28,38 @@
       subject.reject!
       expect(subject).to be_rejected
     end
+
+    it 'is withdrawn after cancellation if not accpeted' do
+      subject.cancel!
+      expect(subject).to be_withdrawn
+    end
+
+    it 'is cancelled after cancellation if accepted' do
+      subject.accept!
+      subject.cancel!
+      expect(subject).to be_cancelled
+    end
+
+    it 'exposes booked as not processable' do
```

Comment:
These names feel a bit passive-voice-y to me: `it 'exposes booked as not processable'` is a bit… odd? Maybe `it 'is not processable after booking'` and so on?